arroyo-api/src/sources.rs

```diff
@@ -196,6 +196,21 @@ pub enum PrimitiveType {
     UnixMillis,
 }
 
+fn primitive_to_sql(primitive_type: &PrimitiveType) -> &'static str {
+    match primitive_type {
+        PrimitiveType::Int32 => "INTEGER",
+        PrimitiveType::Int64 => "BIGINT",
+        PrimitiveType::UInt32 => "UNSIGNED INTEGER", // not really sql
+        PrimitiveType::UInt64 => "UNSIGNED BIGINT",  // not really sql
+        PrimitiveType::F32 => "FLOAT",
+        PrimitiveType::F64 => "DOUBLE",
+        PrimitiveType::Bool => "BOOLEAN",
+        PrimitiveType::String => "VARCHAR",
```

Comment:
I kinda prefer `TEXT` to `VARCHAR`, if we're gonna pick one.

Reply:
Agreed (should have said that in the original issue)